library/psa_crypto_slot_management.c

```diff
@@ -227,6 +227,21 @@ psa_status_t psa_validate_key_persistence( psa_key_lifetime_t lifetime,
 }
 
 #if defined(MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS)
+/** Look up a key identifier as a built-in key.
+ *
+ * \param svc_key_id    The key identifier to look up.
+ * \param[out] slot     On success, a key slot filled with the key metadata
+ *                      and data, including the key identifier.
+ *
+ * \retval #PSA_SUCCESS
+ *         The
```

Comment:
The rest of the sentence is missing.

Reply:
I amended the last commit to finish the sentence.